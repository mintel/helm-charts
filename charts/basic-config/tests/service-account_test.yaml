suite: Test ServiceAccounts
templates:
  - service-accounts.yaml
release:
  namespace: test-namespace
tests:
  - it: Basic Service Account
    set:
      global.name: test-app
      serviceAccount:
        create: true
        name: test
    asserts:
      - isKind:
          of: ServiceAccount
      - hasDocuments:
          count: 1
      - equal:
          path: metadata.namespace
          value: test-namespace
      - equal:
          path: metadata.name
          value: test
