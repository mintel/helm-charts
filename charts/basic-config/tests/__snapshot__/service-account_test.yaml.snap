Basic Service Account:
  1: |
    apiVersion: v1
    automountServiceAccountToken: true
    kind: ServiceAccount
    metadata:
      labels:
        app.kubernetes.io/managed-by: Helm
        app.mintel.com/env: local
        app.mintel.com/region: local
        name: test-app
      name: test
      namespace: test-namespace
